evalml/data_checks/outliers_data_check.py

```diff
@@ -140,7 +140,7 @@ def get_boxplot_data(data_):
             ...                "median": -3.0,
             ...                "q3": -2.0,
             ...                "low_bound": -7.0,
```

Comment:
@tamargrey with the changes should this low bound be `-4.0` instead of `-7.0`?

Reply:
It should be the minimum if low bound is less than the minimum but since the min is -1201 I think it's expected for the low bound to stay at -7?

Reply:
that's correct, thank's Freddy!